src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+
+    abstract void getDraw();
```

Comment:
should be in interface called `Drawable`
```suggestion
```

Reply:
not fixed, read comments carefully before requesting review. feel free to ask questions if something is not understood

Reply:
Plz fix this